app/src/main/java/org/fossasia/openevent/general/auth/EditProfileFragment.kt

```diff
@@ -76,6 +76,11 @@ class EditProfileFragment : Fragment() {
 
         editProfileViewModel.message.observe(this, Observer {
             Toast.makeText(context, it, Toast.LENGTH_LONG).show()
+            editProfileViewModel.isUpdated.observe(this, Observer {
+                it.let {
```

Comment:
This is useless

Reply:
But if I directly use `onBackPressed()` in let operator without observing `isUpdated` in that case if there is an error in updating the profile, then also it direct to profile fragment.

Reply:
The `it.let` is useless

It's like calling 

```java
if (true) {
   onBackPressed()
}
```

Reply:
But in that case 
```java
editProfileViewModel.isUpdated.observe(this, Observer {
                if (it!!) {
```
How can I ignore the use of `!!` and make `it` Nullable?